### PR TITLE
LPS-166236 [EVP] Show the request information

### PR DESCRIPTION
**Pull request by EuniceTravassos**

### CODE REVIEW

**Comment by EuniceTravassos** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layouts/02_my_evp/page-definition.json [hunk 1]
```diff
@@ -3,8 +3,29 @@
 		"pageElements": [
 			{
 				"definition": {
+<<<<<<< HEAD
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layouts/02_my_evp/page-definition.json:6]: There is some conflict resolution text added to this file, please delete them and see if the page is working well

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

**Comment by Claudionilton** [comment]
Just starting reviewing :)

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
Just starting reviewing :)

**Comment by juliocezar96** [comment]
There are 2 repositories: "evp-organization-details", you can put your modifications inside what already exists.

**Comment by EuniceTravassos** [comment]
ci:test:sf

**Comment by EuniceTravassos** [comment]
ci:test:sf

**Comment by EuniceTravassos** [comment]
ci:test:relevant

**Review by silasbsi — APPROVED** [review]
LGTM :heavy_check_mark:

**Comment by juliocezar96** [comment]
ci:test:relevant

**Review by juliocezar96 — APPROVED** [review]
Nice job! 🤙

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-resquest-details/page-definition.json [hunk 2]
```diff
@@ -0,0 +1,1303 @@
+{
+	"pageElement": {
+		"pageElements": [
+			{
+				"definition": {
+					"fragmentStyle": {
+						"borderWidth": "0"
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-resquest-details/page-definition.json:7]: Take this out

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-resquest-details/page-definition.json [hunk 3]
```diff
@@ -0,0 +1,1303 @@
...
+						"opacity": 100,
+						"widthType": "Fluid"
+					}
+				},
+				"pageElements": [
+					{
+						"definition": {
+							"fragmentStyle": {
+								"borderRadius": "borderRadiusSm",
+								"borderWidth": "0",
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-resquest-details/page-definition.json:20]: take this out

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-resquest-details/page-definition.json [hunk 4]
```diff
@@ -0,0 +1,1303 @@
...
+							"fragmentFields": [],
+							"indexed": true
+						},
+						"type": "Fragment"
+					},
+					{
+						"definition": {
+							"fragmentStyle": {
+								"borderRadius": "borderRadiusSm",
+								"borderWidth": "0",
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layout-page-templates/display-page-templates/evp-resquest-details/page-definition.json:189]: take this out

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

**Comment by EuniceTravassos** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @EuniceTravassos

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by silasbsi** [comment]
ci:forward

**Review by Claudionilton — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**